app/views/uniform/many.scala.html

```diff
@@ -107,9 +107,6 @@ <h2 class="heading-small add-to-list" aria-hidden="true">@Messages(s"$key.list-c
                     </legend>
 
             }
-                @if(messages.isDefinedAt(s"$key.list-control.lead")) {
```

Comment:
Why has this been removed?

Reply:
I see why you have now, but the helper could be used by other pages in the future, probably best to just delete the relevant message here

Reply:
Removed the message but retained the tag as suggested.

Reply:
I meant to leave the If statement and span and message in the html. But delete the message we wanted to stop being shown, which is on line 825 of the messages file. That way we can still use this tag if we choose to define a message in future.